packages/services/api/src/modules/operations/providers/traces.ts

```diff
@@ -363,6 +399,7 @@ export class TraceBreakdownLoader {
 
       const query = sql`
         ${sql.join(statements, ' UNION ALL ')}
+        SETTINGS max_threads = 8
```

Comment:
What would be the behaviour without this statement? Does adding this limit the amount of allocated threads or increase the amount of allocated threads used for this query?